addons/point_of_sale/tests/test_frontend.py

```diff
@@ -818,6 +819,22 @@ def test_pos_closing_cash_details(self):
         self.assertEqual(len(pos_session.statement_line_ids), 1)
         self.assertEqual(pos_session.statement_line_ids[0].amount, -10)
 
+    def test_pos_closing_cash_decimals(self):
+        """
+        Test that the closing cash difference is rounded according to the currency specification
+        and does not have excessive decimal places.
+        """
+        self.main_pos_config.open_ui()
+        current_session = self.main_pos_config.current_session_id
+        current_session.post_closing_cash_details(560.40)
+        current_session.close_session_from_ui()
+
+        self.main_pos_config.with_user(self.pos_user).open_ui()
+        self.start_tour("/pos/ui?config_id=%d" % self.main_pos_config.id, 'CashClosingDecimals', login="pos_user")
+
+        pos_session = self.main_pos_config.current_session_id
+        self.assertEqual(pos_session.message_ids[0].body, Markup('<p>Opening difference: $&nbsp;-\ufeff1.91<br></p>'))
```

Comment:
```suggestion
self.assertEqual(pos_session.message_ids[0].body.striptags(), 'Opening difference: $\xa0-\ufeff1.91')
```

is it working just with this ?

Reply:
yes, it is :)